nodes.erl

```diff
@@ -73,7 +78,7 @@ query_to_commit(OtherNodes) ->
     broadcast(OtherNodes, {query, self()}).
 
 broadcast(Nodes, Message) ->
-    lists:map(fun(Node) -> Node ! Message end, Nodes).
+    [Node ! Message || Node <- Nodes].
```

Comment:
so that's how broadcast is done! I'm learning comprehensions from Elixir and now I understand